src/Core/SharedKernels/Questionnaire/WB.Core.SharedKernels.Questionnaire/Documents/Question/DateTimeQuestion.cs

```diff
@@ -20,7 +22,7 @@ public override void AddAnswer(Answer answer)
 
         public override IEnumerable<T> Find<T>(Func<T, bool> condition)
         {
-            return new T[0];
+            return Array.Empty<T>();

```

Comment:
need remove this method and need inhered this class from 'ExternalServiceQuestion'